### PR TITLE
fix(conversation-summary): adding conversation summary card

### PR DESCRIPTION
**Pull request by ceciliamariar**
Adding conversation summary card

image

### CODE REVIEW

File: src/components/ConversationSummary.vue [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+      default: 532
+    },
+    disableLink: {
+      type: Boolean,
+      default: false
+    },
+    showMoreMsg: {
+      type: String,
+      default: 'Ver mais'
+    },
```
Comment (VMarchena) [inline src/components/ConversationSummary.vue:94]: Are we using all these properties?

File: src/components/BlipGroupCard.vue [hunk 2]
```diff
@@ -168,6 +169,20 @@ export default {
...
+  methods: {
+    isFailedMessageGroup(message, group) {
+      if (message.type === MessageTypesConstants.CONVERSATION_SUMMARY) {
+        return ''
+      }
+      return message.status === 'failed' && message.position === 'right' && group.hasNotification ? ' failed-message' : ''
+    },
+    failedMessageNotification(type) {
+      if (type === MessageTypesConstants.CONVERSATION_SUMMARY) {
+        return this.translations.failedToLoadConversationSummary
```
Comment (VMarchena) [inline src/components/BlipGroupCard.vue:182]: Shouldn't it be the responsibility of the `ConversationSummary` component to display this `failedToLoadConversationSummary`?

**Review by leonardo-avila — APPROVED** [review]

File: src/components/ConversationSummary.vue [hunk 3]
```diff
@@ -0,0 +1,190 @@
+<template>
+  <div class="blip-container summary">
+    <div :class="'bubble ' + position">
+      <div v-if="document.isLoading">
+        <div class="typing">
+          <div class="typing-dot"></div>
+          <div class="typing-dot"></div>
+          <div class="typing-dot"></div>
+        </div>
+      </div>
```
Comment (VMarchena) [inline src/components/ConversationSummary.vue:10]: I think we have the same loading behavior on component `ChatState`. Do you think it makes sense to create a component for this loading and use it in both components?

File: src/components/ConversationSummary.vue [hunk 4]
```diff
@@ -0,0 +1,190 @@
...
+      default: false
+    },
+    showMoreMsg: {
+      type: String,
+      default: 'Ver mais'
+    },
+    failedToSendMsg: {
+      type: String,
+      default: 'Falha ao enviar a mensagem'
+    }
```
Comment (VMarchena) [inline src/components/ConversationSummary.vue:98]: What is the difference between this `failedToSendMsg` and `failedToLoadConversationSummary` used in the `BlipGroupCard`?

Comment (ceciliamariar) [reply src/components/ConversationSummary.vue:98]: The message content is different
failedToSendMsg  --> Falha ao enviar a mensagem.
failedToLoadConversationSummary  --> Falha ao carregar o resumo da conversa.

**Pull request merged**